policyengine_us/parameters/gov/states/la/tax/income/credits/cdcc/agi_threshold.yaml

```diff
@@ -0,0 +1,12 @@
+description: Louisiana provides a refundable cdcc below this threshold of the federal AGI threshold.
+values:
+  2006-01-01: 25_000
+
+
+metadata:
+ period: year
+ unit: currency-USD
+ label:  Louisiana cdcc refundable threshold
+ reference:
```

Comment:
Please cite the latest tax forms as well, here and elsewhere, as governments often inflation-index dollar values.